src/bibliography.ts

```diff
@@ -11,23 +11,16 @@ import "reflect-metadata";
  * {
  * "title": "My Bibliography",
  * "description": "A collection of references.",
- * "references": [
- *   {
- *     "id": "doe1",
- *     "type": "book",
- *    "title": "The Title",
- *     "author": [
- *       {
- *         "family": "Doe",
- *         "given": "Jane"
- *       }
- *     ],
- *     "issued": "2023"
- *   }
- * ]
- *}
  */
 export class Bibliography {
+	/**
+	 * The references object..
+	 *
+	 * @items.minimum 1
+	 */
+	// TODO get this working with class-transformer
+	// @Type(() => IDReference) // TODO causes and error
```

Comment:
This line generating a confusing error message has me wondering this will work with `class-transformer`.